src/EditorFeatures/Core/Classification/Semantic/AbstractSemanticOrEmbeddedClassificationViewTaggerProvider.cs

```diff
@@ -33,28 +32,17 @@ namespace Microsoft.CodeAnalysis.Classification;
 /// in the editor.  We use a view tagger so that we can only classify what's in view, and not
 /// the whole file.
 /// </summary>
-internal abstract class AbstractSemanticOrEmbeddedClassificationViewTaggerProvider : AsynchronousViewportTaggerProvider<IClassificationTag>
+internal abstract class AbstractSemanticOrEmbeddedClassificationViewTaggerProvider(
+    TaggerHost taggerHost,
+    ClassificationTypeMap typeMap,
+    ClassificationType type)
+    : AsynchronousViewportTaggerProvider<IClassificationTag>(taggerHost, FeatureAttribute.Classification)
 {
-    private readonly ClassificationTypeMap _typeMap;
-    private readonly IGlobalOptionService _globalOptions;
```

Comment:
occasionally, the subclasses would also hold onto state that was passed to the base type.  this was caught now that we have the 'host' object ew pass down.